app/assets/javascripts/global_activities/side_pane.js

```diff
@@ -124,6 +125,34 @@ $(function() {
     });
   }
 
+  function preloadFilters(filters) {
+    updateRunning = true;
+    if (filters.types) {
+      $('.ga-side .activity-selector select').val(filters.types).trigger('change');
+    }
+    if (filters.teams) {
+      $('.ga-side .team-selector select').val(filters.teams).trigger('change');
+    }
+    if (filters.users) {
+      $('.ga-side .user-selector select').val(filters.users).trigger('change');
+    }
+    if (filters.from_date) {
+      $('#calendar-from-date').data('DateTimePicker').date(new Date(filters.from_date));
+    }
+    if (filters.to_date) {
+      $('#calendar-to-date').data('DateTimePicker').date(new Date(filters.to_date));
+    }
```

Comment:
Just double-check: are these 2 `new Date(string)` calls at any point problematic with regards to date conversion due to time zone conversions? I'd guess not, we will always be providing the date as `YYYY-MM-DD`, which should be future-proof, right?

Reply:
Aleksej use this format on backend, so i also use it on front end. I'll prefer to use everywhere unix GMT time and convert it only on front-end to user time zone.